### PR TITLE
Fix wrapped text position when change textbounds on scaled parent

### PR DESCRIPTION
**Pull request by reatailret**

### CODE REVIEW

**Comment by samme** [comment]
Can you add a note in [README.md: Change Log: Unreleased](https://github.com/photonstorm/phaser-ce/blob/master/README.md#unreleased)?

**Comment by reatailret** [comment]
Sorry, I do not know how to do this. Maybe you will add?

**Comment by gale93** [comment]
@reatailret You just need to modify the readme file under Unreleased / bug fixes and update the pull request.
[You can do it easily (hint)](https://stackoverflow.com/questions/10147445/github-adding-commits-to-existing-pull-request)

**Comment by bert198211** [comment]
Tz.dmveritig

File: src/gameobjects/Text.js [hunk 1]
```diff
@@ -1421,7 +1421,7 @@ Phaser.Text.prototype.setTextBounds = function (x, y, width, height) {
             this.style.wordWrapWidth = width;
         }
     }
-
+	this.updateText();
```
Comment (Nek-) [inline src/gameobjects/Text.js]: Can you fix the indentation please ?

**Pull request closed without merging**